.github/workflows/ci.yaml

```
@@ -8,59 +8,72 @@
-      with:
-        python-version: ${{ matrix.python-version }}
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
 
-    - name: Install the project
-      run: uv sync --all-extras --dev
+      - name: Install development dependencies
+        run: pip install .[dev]
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)